src/components/ScholarshipList.jsx

```diff
@@ -1,125 +1,53 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
-import {
-  Button,
-  Card,
-  CardActionArea,
-  CardActions,
-  CardContent,
-  CardHeader,
-  Grid,
-  IconButton,
-  Link as MuiLink,
-  Typography,
-} from '@material-ui/core';
-import BookmarkBorderIcon from '@material-ui/icons/BookmarkBorder';
-import ShareIcon from '@material-ui/icons/Share';
-import { makeStyles } from '@material-ui/core/styles';
+import { Grid } from '@material-ui/core';
 import PropTypes from 'prop-types';
 import { BRAND_NAME } from '../config/constants';
 import ShareDialog from './ShareDialog';
 
-const useStyles = makeStyles((theme) => ({
-  actions: {
-    padding: theme.spacing(2),
-    paddingTop: theme.spacing(1),
-  },
-  content: {
-    paddingTop: theme.spacing(0),
-    paddingBottom: theme.spacing(1),
-  },
-  description: {
-    display: '-webkit-box',
-    '-webkit-line-clamp': 5,
-    lineClamp: 5,
-    '-webkit-box-orient': 'vertical',
-    overflow: 'hidden',
-    whiteSpace: 'pre-line',
-  },
-}));
+import ScholarshipListCard from './ScholarshipListCard';
+
+// move out from component
+// because it don't need to recreate every time component render
```

Comment:
Maybe leave this how it was? The [build](https://github.com/beyondhb1079/s4us/pull/369/checks?check_run_id=1815635718) is complaining about unused variables. You can add `// eslint-disable-next-line no-unused-vars` above the declaration or comment this out as it's going to be reused eventually somewhere else in the codebase.